src/open_producten/producttypen/views.py

```diff
@@ -0,0 +1,66 @@
+from django_filters.rest_framework import DjangoFilterBackend
+from rest_framework.decorators import action
+from rest_framework.response import Response
+
+from open_producten.producttypen.models import (
+    Link,
+    Onderwerp,
+    Prijs,
+    ProductType,
+    Vraag,
+)
+from open_producten.producttypen.serializers import (
+    LinkSerializer,
+    OnderwerpSerializer,
+    PrijsSerializer,
+    ProductTypeActuelePrijsSerializer,
+    ProductTypeSerializer,
+    VraagSerializer,
+)
+from open_producten.utils.views import OrderedModelViewSet
+
+
+class ProductTypeViewSet(OrderedModelViewSet):
+    queryset = ProductType.objects.all()
+    serializer_class = ProductTypeSerializer
+    lookup_url_kwarg = "id"
+
+    @action(
+        detail=False,
+        serializer_class=ProductTypeActuelePrijsSerializer,
+        url_path="actuele-prijzen",
+    )
+    def actuele_prijzen(self, request):
+        product_typen = ProductType.objects.all()
```

Comment:
```suggestion
        product_typen = self.get_queryset().all()
```

Use the query set method here